comeet/UI/RoomsList/RoomsListViewController.swift

```diff
@@ -65,6 +65,13 @@ class RoomsListViewController: BaseViewController {
         endTimelabel.text = "End " + end
     }
     
+    func hideGuideView() {
+        if ( self.guideView.isHidden == false) {
+            self.guideView.isHidden = true
+        }
```

Comment:
You can set this unconditionally, unless doing so will make the UI flicker or something.